docs/src/implemented-proposals/rpc-transaction-history.md

```diff
@@ -106,3 +106,13 @@ This table maps a transaction signature to its confirmed block, and index within
 
 The row key is the base58-encoded transaction signature.
 The row data is a compressed `TransactionInfo` struct.
+
+### Entries Table: `entries`
```

Comment:
Do you think we should date this piece to make it explicitly clear that this wasn't implemented from the start / isn't going to be available for all historical data unless operators manually backfill their tables ?

Reply:
Yes, that makes complete sense. I'll add in the version number.

Reply:
🔥🎁👍🪃🐉